### DISCUSSION TITLE
Copilot chat shows then hide on VS Code

### DISCUSSION THREAD

**Discussion by mhore22** [community/community#88480] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have a copilot subscription from organization and it's enabled.

![image](https://github.com/community/community/assets/12798131/aa6f14f6-0646-435c-9961-d14c80ae45d2)

Now when I login to Github on VSCode the chat icon suddenly disappears
![image](https://github.com/community/community/assets/12798131/1c4713ee-4cf7-431a-b690-e284f479cf0a)

Copilot status is ready
![image](https://github.com/community/community/assets/12798131/2473469c-b621-4f3c-86e6-2c8812cfb933)

I'm not sure how to fix this.

**Comment by dmonder** [comment]
This exact thing is happening to me, and I cannot figure it out. It is driving me nuts! I am able to use Copilot just fine but Copilot Chat is not available.

**Reply by sandyatkinson-rentall** [reply]
yes, I'm having this issue as well.

**Reply by mhore22** [reply, marked as answer]
Hi, after a long troubleshooting.

I asked the admin on my organization to enable the Copilot chat on the organization level.

Here's the link to enable the Copilot chat(https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#enabling-or-disabling-github-copilot-chat-at-the-organization-level)

Once it's enabled, go to you profile copilot settings and you should see that "**Copilot Chat in the IDE**" is enabled.
![image](https://github.com/community/community/assets/12798131/5b36e7ff-5228-4c86-8c48-594c6da78744)

Hope this also solve your problem.

**Reply by dmonder** [reply]
That did it. Thank you so much for the help.
